Update template, rerender

pmlandwehr opened the pull request.

pmlandwehr committed:
Update template, add license info

pmlandwehr committed:
MNT: Re-rendered with conda-smithy 1.3.3

pmlandwehr committed:
Tick build number

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (`recipe`) and found it was in an excellent condition.

pmlandwehr committed:
Added pyqt to run reqs

The windows build is unhappy because it's missing the pyqt backend. Added pyqt as a requirement on the theory that it'll help with the CI. We'll see...

pmlandwehr committed:
Specify qt and pyqt range

pmlandwehr committed:
Update meta.yaml
